test/client/data/dom-utils/iframe.html

```diff
@@ -14,10 +14,10 @@
 </head>
 <body>
 <script type="text/javascript">
-    var HH_SETTINGS = Hammerhead.get('./settings');
-    HH_SETTINGS.set({
-        CROSS_DOMAIN_PROXY_PORT: 2000
-    });
+    var hammerhead = window['%hammerhead%'];
+    var settings = hammerhead.get('./settings');
+
+    settings.get().crossDomainProxyPort = '5555';
```

Comment:
and why in the `test/client/data/focus-blur-change/iframe.html` you do it in the other way? https://github.com/DevExpress/testcafe/pull/187/files#diff-1cdeefba9b0d2cd285052d0c612fd36fR15
Maybe we should do the same here?